src/screens/MainUserPage/MainUserPage.module.css

```diff
@@ -0,0 +1,327 @@
+.wrappage {
```

Comment:
Although this is an extreme case, please apply a flex-direction: column, to the interests segment, when the device is below a certain width/media breakpoint, as the interest bubbles overflow horizontally as the page is resized

![image](https://user-images.githubusercontent.com/55003601/125209182-c7affc00-e25c-11eb-9749-341a110aacb7.png)

Reply:
Will do that.